x-pack/legacy/plugins/reporting/server/lib/iso_string_validate.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const isoStringValidate = (input: string): string | undefined => {
+  const message = `value must be a valid ISO format`;
```

Comment:
NIT: `message` could be extracted outside of `isoStringValidate`body. Upsides are insignificant though as it's exported as static.